appveyor.yml

```diff
@@ -4,4 +4,4 @@ environment:
   GOPATH: c:\gopath
 stack: go 1.14
 test_script:
-  - go test
+  - go test --godog.tags=windows
```

Comment:
Should this be squashed with the commit which added the @linux,@windows, ... tags?

Reply:
This is specific to CI so I have as separate commit.